Stack/Opc.Ua.Core/Types/Utils/Utils.cs

```diff
@@ -2790,6 +2790,39 @@ public static X509Certificate2Collection ParseCertificateChainBlob(ReadOnlyMemor
             return certificateChain;
         }
 
+
+        /// <summary>
+        /// Creates a DER blob from a X509Certificate2Collection.
+        /// </summary>
+        /// <param name="certificates">The certificates to be returned as raw data.</param>
+        /// <returns>
+        /// A DER blob containing zero or more certificates.
+        /// </returns>
+        public static byte[] CreateCertificateChainBlob(X509Certificate2Collection certificates)
```

Comment:
Can we have a unit test for this method? We could, e.g., replace the code which creates the chain in the test ParseCertificateBlob() in CertificateFactoryTest.cs with a call to this method. Does that make sense?

Reply:
good idea